docs/operations/es_reader.md

```diff
@@ -0,0 +1,180 @@
+
```

Comment:
The filename here should be same the processor, `elasticsearch_reader`. Also there is some whitespace at the top of file that should be removed.